### ISSUE TITLE
Play/Pause buttons missing, even in demo

### ISSUE THREAD

**Issue by wimhuiskes** [tylerpearson/lazyYT#51]
Like the title says. Just recently the Play/Pause buttons are missing.

**Comment by mgav** [comment]
Hello @wimhuiskes  
I'm a novice, but I believe you can display/hide these controls using the parameter controls=1  
Scroll down this page to learn more: https://developers.google.com/youtube/player_parameters 
Best of luck. -Mark

**Comment by wimhuiskes** [comment]
Hello @mgav 

Problem is probably Firefox, because it doenst happen in Chrome or Safari.

**Comment by mgav** [comment]
Thank you @wimhuiskes - I think that's almost worse b/c the effort to fix isn't richly rewarded, since most browsers aren't affected.  Best of luck! -mg

**Comment by ghost** [comment]
Check your Firefox plugins, Ghostery and Adblocker are the usual suspects.

**Comment by Daugilas** [comment]
Yes it seems to be FF problem as I also have the same issue with regular YouTube embeds.

btw.: check the latest version of this plugin: https://github.com/Daugilas/lazyYT
